mcs/class/System.Data/System.Data-net_4_x.csproj

```diff
@@ -49,6 +49,26 @@
   </PropertyGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <ItemGroup>
+    <Compile Include="..\..\..\external\corefx\src\System.Data.Common\src\System\Data\SQLTypes\INullable.cs" />

```

Comment:
@akoeplinger I didn't as far as I remember, 'make update-solution-files' did it :)